src/commands/utility/changeStatus.js

```diff
@@ -0,0 +1,55 @@
+const { SlashCommandBuilder } = require("discord.js");
+
+const MAPPED_STATUS_COMMANDS = {
+  'pr-no-merge': '🚫',
+  'pr-sos': '🆘',
+  'pr-draft': '🚧',
+  'pr-reviewing': '👀',
+  'pr-request-changes': '🔁',
+  'pr-request-review': '❗',
+  'pr-working-in-fixes': '🧑‍🔧',
+  'pr-approved': '✅',
+  'pr-merged': '✅🛫',
+  'pr-merged-need-tasks': '✅🛫📝',
+  'pr-merged-task-created': '✅🛫📋'
+};
+
+const COMMAND_KEYS = Object.keys(MAPPED_STATUS_COMMANDS);
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('changestatus')
+    .setDescription('Change the status of the thread')
+    .addStringOption(option =>
+      option.setName('status')
+        .setDescription('status command')
+        .setRequired(true)
+        .addChoices(
+          COMMAND_KEYS.map(command => ({ name: `/${command}`, value: command }))
+        )
+    ),
+  async execute(interaction) {
+    try {
+
+      const { options, channel } = interaction;
+
+      if (!channel.isThread()) return await interaction.reply("Sorry, this is not a thread!");
```

Comment:
I think there is a way to make discord say this only for the executor of the command, i think with the ephemeral property
![image](https://github.com/user-attachments/assets/ffc5be9e-e677-49f6-be86-440d6c23a722)

Reply:
It must be applied to all replies or only for that?

Reply:
I think the replies that all replies

Reply:
I changed my mind. This one seems like a feedback that would be interesting for others. But you could change the emojis for the name of the command. For example, instead of: `Status updated to ❗🛫📝 Test`. You could say `Status updated to request review`
![image](https://github.com/user-attachments/assets/bca96e0b-e518-48af-99c7-29d3a23e84d6)

Maybe others doesn't knows what the emojis mean, but the Discord Bot message explain a bit what it means so they can understand easily

Reply:
![image](https://github.com/user-attachments/assets/088359df-f0f5-4c4e-bbd1-b148865e3f7d)

Reply:
ok! Sounds good for me